src/analysisd/expression.c

```diff
@@ -155,9 +155,10 @@ bool w_expression_match(w_expression_t * expression, const char * str_test, cons
     bool retval = false;
     const char * ret_match = NULL;
 
-    regex_matching status_match = {0};
-    pcre2_match_data * match_data;
-    PCRE2_SIZE * ovector;
+    regex_matching status_match;
+    memset(&status_match, 0, sizeof(status_match));
```

Comment:
That is correct, but this allows zeroing the data at compile time:
```c
regex_matching status_match = { .sub_strings = NULL };
```
(Setting the first member to 0 forces the compiler to write the entire structure).

Reply:
Fixed